nima/webclient/api/src/main/java/io/helidon/nima/webclient/api/Proxy.java

```diff
@@ -429,14 +447,22 @@ private static Socket connectToProxy(WebClient webClient,
                                                                     })
                 .connect();
 
-        HttpClientResponse response = webClient.method(Http.Method.CONNECT)
+        HttpClientRequest request = webClient.method(Http.Method.CONNECT)
                 .followRedirects(false) // do not follow redirects for proxy connect itself
                 .connection(connection)
                 .uri("http://" + proxyAddress.getHostName() + ":" + proxyAddress.getPort())
                 .protocolId("http/1.1") // MUST be 1.1, if not available, proxy connection will fail
                 .header(Http.Header.HOST, targetAddress.getHostName() + ":" + targetAddress.getPort())
-                .accept(MediaTypes.WILDCARD)
-                .request(); // we cannot close the response, as that would close the connection
+                .accept(MediaTypes.WILDCARD);
+        if (clientConfig.keepAlive()) {
+            request.header(Http.HeaderValues.CONNECTION_KEEP_ALIVE)
+            .header(PROXY_CONNECTION);
```

Comment:
The formatting is not OK. If you use intellij idea with the formatting provided in etc (of the project), you should be able to just reformat the piece of code and it will be OK. Otherwise check another class that has multiline fluent API usage and align it with that please

Reply:
I took the line 456 as an example. I think now it is correct.